src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/CreateNewListModalWizard.tsx

```diff
@@ -1,32 +1,31 @@
 import { FC, useState } from "react"
-import { CreateNewListModal, NewAddedList } from "./CreateNewListModal"
+import { CreateNewListModal, ArtworkList } from "./CreateNewListModal"
 import { AddArtworksModal } from "./AddArtworksModal"
 
 export interface CreateNewListModalWizardProps {
-  onComplete: () => void
+  onComplete: (artworkList: ArtworkList) => void
   onClose: () => void
 }
 
 export const CreateNewListModalWizard: FC<CreateNewListModalWizardProps> = ({
   onComplete,
   onClose,
 }) => {
-  const [listName, setListName] = useState<string | null>(null)
+  const [artworkList, setArtworkList] = useState<ArtworkList | null>(null)
 
-  const handleCreateListComplete = (list: NewAddedList) => {
-    setListName(list.name)
+  const handleCreateListComplete = (list: ArtworkList) => {
+    setArtworkList(list)
   }
 
-  const handleAddArtworksComplete = () => {
-    onComplete()
+  const handleAddArtworksComplete = (list: ArtworkList) => {
```

Comment:
**suggestion**: we already store the created list in `useState` and we don't have to pass it as an argument. Feel free to ignore if you don't like the proposed changes.

```javascript
const handleAddArtworksComplete = () => {
  onComplete(artworkList!)
}
```